src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -511,6 +511,26 @@ user:
         delete_confirm: "Voulez-vous vraiment ?"
         back_to_list: "Revenir à la liste"
 
+site_credential:
+    page_title: Gestion des accès aux sites
+    new_site_credential: Créer un accès à un site
+    edit_site_credential: Éditer l'accès d'un site
+    description: "Ici vous pouvez gérer les accès aux différents sites. Ces accès permettent de récupérer des contenus sur des sites qui requiert une authentification ou un paywall"
+    list:
+        actions: Actions
+        edit_action: Éditer
+        yes: Oui
+        no: Non
+        create_new_one: Créer un nouvel accès à un site
+    form:
+        username_label: 'Identifiant'
+        host_label: 'Domaine'
+        password_label: 'Mot de passe'
+        save: "Sauvegarder"
+        delete: "Supprimer"
+        delete_confirm: "Voulez-vous vraiment ?"
```

Comment:
"Voulez-vous vraiment supprimer ?" or "Êtes vous sur ?". But "Voulez-vous vraiment ?" seems strange to me.